src/modules/navigator/rtl.h

```diff
@@ -91,13 +94,15 @@ class RTL : public MissionBlock, public ModuleParams
 		RTL_STATE_LANDED,
 	} _rtl_state{RTL_STATE_NONE};
 
+	float _rtl_alt;	// AMSL altitude at which the vehicle should return to the home position
```

Comment:
```suggestion
	float _rtl_alt{0.0f};	// AMSL altitude at which the vehicle should return to the home position
```

Reply:
Agreed.

Reply:
@julianoes Thanks, I fixed that before but then it got lost during a reset. Fixed now!